src/lib/CommandHandler.ts

```diff
@@ -114,30 +114,19 @@ export class CommandHandler {
         });
     }
 
-    protected resolveCommandToInputId(cmd: string | undefined) {
+    protected resolveCommandToInputId(cmd: string | undefined, desc: string | undefined) {
         // Lookup the inputId from the supplied command input string
         if (!cmd) return undefined;
 
         let inputs: any[] = [];
-        if (vscode.workspace.workspaceFolders) {
```

Comment:
did you test this with multiple folder workspace?

Reply:
test
    -> folder1
       -> tasks.json
    -> folder2
       -> tasks.json
    tasks.json
    test.code-workspace
 
I put two subfolder in the same directory with .code-workspace file. For each subfolder and for project folder I created tasks.json files. And those tasks is not recognized even by vscode(not visible in Tasks: Run Task). So I think it is not necessary to scan explicitly those folders. inspect() api gets all tasks from all scopes if it is accessible within the rules of vscode. I tried also to open the same directory without using .code-workspace file. Only the workspace tasks and user tasks are visible(valid). There are 6 or 7 members in the returned inputs including global and workspace. But these are the only ones I can observe some values.

Reply:
Sadly it does not work as before. Check my example @ https://github.com/augustocdias/vscode-shell-command/issues/63

I'll have a look